internal/app/cf-terraforming/cmd/generate.go

```diff
@@ -1068,6 +1068,23 @@ func generateResources() func(cmd *cobra.Command, args []string) {
 				// tls_1_2_only is deprecated in favour of min_tls
 				jsonStructData[i].(map[string]interface{})["settings"].(map[string]interface{})["tls_1_2_only"] = nil
 			}
+		case "cloudflare_tiered_cache":
+			tieredCache, err := api.GetTieredCache(context.Background(), &cloudflare.ResourceContainer{Identifier: zoneID})
+			if err != nil {
+				log.Fatal(err)
+			}
+			var jsonPayload []cloudflare.TieredCache
+			jsonPayload = append(jsonPayload, tieredCache)
+
+			resourceCount = 1
+			m, _ := json.Marshal(jsonPayload)
+			err = json.Unmarshal(m, &jsonStructData)
+			if err != nil {
+				log.Fatal(err)
+			}
+
+			jsonStructData[0].(map[string]interface{})["id"] = zoneID
```

Comment:
you shouldn't need to set the ID since it isn't used.

Reply:
As mentioned in the last comment, I was following the provider example, if that's not needed I'll remove it.

Reply:
zone settings are a little weird and need this. i'll remove it.